resources/translations.ts

```diff
@@ -196,6 +196,9 @@ export const translateWithReplacements = (
   replaceLang: Lang,
   replacements?: TimelineReplacement[],
 ): { text: string; wasTranslated: boolean } => {
+  if (text === '')
+    return { text: '', wasTranslated: true };
```

Comment:
When does this happen?

Reply:
We use `''` in some trigger to match all.

Reply:
I looked but can't find them with a search.  Can you point out where?

Reply:
https://github.com/quisquous/cactbot/blob/c728fb9ab1e3dd26334a98d90d6786712323de04/ui/raidboss/data/04-sb/raid/o10s.ts#L106

Reply:
emmm it's not a match all, just match blank.

Reply:
Ok ok, thanks.  I remember that now.